app/components/ConstructorStats.tsx

```diff
@@ -59,7 +59,7 @@ const PuzzleLink = ({
   }
   return (
     <li>
-      <Link href={'/crosswords/' + puzzleId}>
+      <Link href={`/crosswords/${puzzleId}/${slugify(puzzle?.t)}`}>
```

Comment:
Does `slugify` handle nulls/undefineds? I don't think it does so I'm surprised this isn't a type error. If I'm right then we should just add an `|| ''` here so the link will just have an empty slug param until puzzle exists